charts/generic/pod-cpu-hog/experiment.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: litmuschaos.io/v1alpha1
+description:
+  message: |
+    Injects cpu consumption on pods belonging to an app deployment
+kind: ChaosExperiment
+metadata:
+  name: pod-cpu-hog
+  version: 0.1.0
+spec:
+  definition:
+    scope: Namespaced
+    permissions:
+      apiGroups:
+        - ""
+        - "batch"
+        - "litmuschaos.io"
+      resources:
+        - "jobs"
+        - "pods"
+        - "pods/exec"
+        - "chaosengines"
```

Comment:
Is `pods/exec` is required in resources ? @ksatchit

Reply:
Found that it is not needed. There are `docker exec` steps being performed in this experiment, but that doesn't come under the scope of K8s svcAccount. Thanks !